driftbase/systesthelper.py

```diff
@@ -1,7 +1,7 @@
 from drift.systesthelper import DriftTestCase
 
 
-class DriftBaseTestCase(DriftTestCase):
+class DriftTestCase(DriftTestCase):
```

Comment:
This shadowing somehow tickles me wrong.
we could monkeypatch at import time instead, but I don't know if that's more proper.
`
from drift.systesthelper import DriftTestCase
DriftTestCase.auth_provider = 'user+pass'
`

Reply:
Nah this is wrong. The name in drift was bad so I changed it. For some reason I didn't notice it had the same name here. The local test class should be renamed too.